src/uct/api/uct.h

```diff
@@ -693,11 +693,16 @@ ucs_status_t uct_iface_get_address(uct_iface_h iface, uct_iface_addr_t *addr);
  * it does not detect issues such as network mis-configuration or lack of connectivity.
  *
  * @param [in]  iface      Interface to check reachability from.
- * @param [in]  addr       Address to check reachability to.
+ * @param [in]  dev_addr   Device address to check reachability to. It may be NULL
+ *                         if iface_attr.dev_addr_len == 0, and has to be non-NULL otherwise.
+ * @param [in]  iface_addr Interface address to check reachability to. It may
```

Comment:
@shamisp Yes, you are right. Would you like me to change the documentation in the way like:
`Device address to check reachability to. It is NULL if iface_attr.dev_addr_len == 0, and must to be non-NULL otherwise.`